### PR TITLE
Bump version to 19.03.6.rc2

### PR DESCRIPTION
**Pull request by arkodg**

### CODE REVIEW

**Comment by cpuguy83** [comment]
There's a buildkit commit just merged in moby/moby's 19.03 brach. Did you want to pick that up for rc2?

**Comment by arkodg** [comment]
sure @cpuguy83

**Comment by arkodg** [comment]
are we good to go @tiborvass @thaJeztah @tonistiigi @cpuguy83 @AkihiroSuda

**Comment by arkodg** [comment]
good catch @tiborvass, edited the `CHANGELOG`

**Comment by tiborvass** [comment]
You missed https://github.com/moby/moby/pull/40440 (fixes https://github.com/moby/moby/issues/40335) in the Changelog. Something like: `Fixes issue with directory permissions when BuildKit and usernamespaces are enabled`

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ For official release notes for Docker Engine CE and Docker Engine EE, visit the
 
 ### Runtime
 
-* Bump Golang 1.12.15. [docker/engine#452](https://github.com/docker/engine/pull/452)
-* Update containerd binary to v1.2.11. [docker/engine#428](https://github.com/docker/engine/pull/428)
-* Update to runc v1.0.0-rc9. [docker/engine#428](https://github.com/docker/engine/pull/428)
+* Bump Golang 1.12.16. [moby/moby#40433](https://github.com/moby/moby/pull/40433)
+* Update containerd binary to v1.2.12. [moby/moby#40433](https://github.com/moby/moby/pull/40453)
+* Update to runc v1.0.0-rc10. [moby/moby#40433](https://github.com/moby/moby/pull/40453)
 - Fix possible runtime panic in Lgetxattr. [docker/engine#454](https://github.com/docker/engine/pull/454)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Should we mention the CVE's that were fixed both in Golang and Runc/containerd ?

Comment (arkodg) [reply CHANGELOG.md]: From a quick scan of the CHANGELOG, I don't think we've specified Golang CVE's in the past, the links should have the CVE details in there

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,13 +3,14 @@
...
-## 19.03.6 (2020-01-23)
+## 19.03.6 (2020-02-05)
 
 ### Builder
 
 - builder-next: Allow modern sign hashes for ssh forwarding. [docker/engine#453](https://github.com/docker/engine/pull/453)
 - builder-next: Clear onbuild rules after triggering. [docker/engine#453](https://github.com/docker/engine/pull/453)
-- bump hcsshim to fix docker build failing on Windows 1903. [docker/engine#429](https://github.com/docker/engine/pull/429)
+- Bump hcsshim to fix docker build failing on Windows 1903. [docker/engine#429](https://github.com/docker/engine/pull/429)
+- Fix issue with directory permissions when BuildKit and usernamespaces are enabled. [moby/moby#40440](https://github.com/moby/moby/pull/40440)
```
Comment (tiborvass) [inline CHANGELOG.md]: @arkodg sorry didn't see the `builder-next:` prefix. This one should have it as well, and we can thus `s/BuildKit and //` and `s/are enabled/is enabled/`

**Comment by AkihiroSuda** [comment]
Can we have the tag on Moby repo?

**Comment by cpuguy83** [comment]
@AkihiroSuda Yeah we discussed this on slack and seems like we want to go ahead move all the tags in docker/engine.

**Pull request merged**